test/jdk/java/security/spec/TestNamedParameterSpec.java

```diff
@@ -41,6 +41,7 @@ public class TestNamedParameterSpec {
         "ED25519", "ED448", "X25519", "X448",
         "ML_DSA_44", "ML_DSA_65", "ML_DSA_87",
         "ML_KEM_512", "ML_KEM_768", "ML_KEM_1024",
+        "X25519MLKEM768", "SecP256r1MLKEM768", "SecP384r1MLKEM1024",
```

Comment:
Does it sound like an option to use the similar naming style like "ML_KEM_512" for the new names?  for example, X25519MLKEM768 -> X25519_ML_KEM_768

Reply:
We don't need to change `TestNamedParameterSpec.java` test now as the `NamedParameterSpec` instances for hybrid named groups are defined in `Hybrid.java` as internal constants. We use a "two-part" naming style (left and right algorithm), such as `X25519_MLKEM768`. It seems cleaner and more readable than breaking it into multiple parts with additional underscores.